packages/generators/src/utils/run-prettier.ts

```diff
@@ -21,9 +21,7 @@ export function runPrettier(outputPath: string, source: string): void {
         // eslint-disable-next-line node/no-extraneous-require
         prettier = require('prettier');
     } catch (err) {
-        logger.warn(
-            "File is not properly formatted because you don't have prettier installed, you can either install it or format it manually",
-        );
+        logger.warn(`${fileName} is not formatted because prettier is not installed locally.`);
```

Comment:
Prettier can be installed globally :smile:

Reply:
Tried it but for some reason it's not doing a global lookup 🤔 

![image](https://user-images.githubusercontent.com/21009455/107120707-bed58b80-68b4-11eb-8193-2cf34077ecac.png)

Reply:
hm, yep, we need other module for this, maybe we can improve it in future